tests/factories/real-world/reports/create-family-household-survey.js

```diff
@@ -9,7 +9,7 @@ module.exports = new Factory()
     }
     return Faker.faker.helpers.uniqueArray(
       ['boiling', 'filtration', 'chlorination', 'other'],
-      Faker.faker.datatype.number({ min: 1, max: 4 })
+      Faker.faker.number.int({ min: 1, max: 4 })
     ).join(' ');
   })
   .attr('kitchen', Faker.faker.datatype.boolean)
```

Comment:
According to [the documentation](https://fakerjs.dev/api/datatype.html#boolean) the `faker.datatype.boolean()` is a method that returns the boolean value true or false. 

Can you please change the places where the `datatype.boolean` is not used as a method?

I found these instances:
![image](https://github.com/medic/cht-core/assets/94494491/7734e990-9b99-487d-a5e0-aff68a675419)